operations/src/main/java/edu/unc/lib/boxc/operations/impl/metadata/ExportDominoMetadataService.java

```diff
@@ -153,6 +165,45 @@ private void assertParentRecordValid(PID pid, ContentObjectRecord parentRec) {
         }
     }
 
+    private String getContentType(ContentObjectRecord record, AccessGroupSet principals) {
+        // Check for viewable files: video, audio, or image
+        if (accessCopiesService.hasViewableFiles(record, principals)) {
+            var origDatastream = record.getDatastreamObject(DatastreamType.ORIGINAL_FILE.getId());
+            var mimetype = origDatastream.getMimetype();
+            if (mimetype.contains(AUDIO)) {
+                return AUDIO;
+            }
+            if (mimetype.contains(VIDEO)) {
+                return VIDEO;
+            }
+            if (mimetype.contains(IMAGE)) {
+                return IMAGE;
+            }
+        }
+
+        // check for streaming content: audio or video
+        var streamingChild = accessCopiesService.getFirstStreamingChild(record, principals);
+        if (streamingChild != null) {
+            var streamingType = streamingChild.getStreamingType();
+            if (streamingType.contains(SOUND)) {
+                return "streaming audio";
```

Comment:
I know they are each only used once, but for consistency and readability I think i'd be good to have constants for all the content types together at the top.